roles/aap_setup_install/tasks/main.yml

```
@@ -3,11 +3,11 @@
 
 - name: Check Ansible Tower Running
   uri:
-    url: "https://{{ controller_hostname }}/api/v2/ping/"
+    url: "https://{{ controller_hostname | default((aap_setup_prep_inv_nodes['automationcontroller'].keys() | list)[0]) }}"
```

Would it make sense to still have `/api/v2/ping/` at the end?

```suggestion
    url: "https://{{ controller_hostname | default((aap_setup_prep_inv_nodes['automationcontroller'].keys() | list)[0]) }}/api/v2/ping/"
```

---

I was about to create an issue, but isnt `aap_setup_prep_inv_nodes['automationcontroller'].keys() | list)[0]` incorrect? 

`aap_setup_prep_inv_nodes['automationcontroller']` is a list, so there are no keys

---

Andrew is right, fixed and updated.